app/components/nextflow_component.html.erb

```diff
@@ -135,7 +137,11 @@
       </div>
     <% end %>
 
-    <button type="submit" class="button button--state-primary button--size-default" data-nextflow--samplesheet-target="submit">
+    <button
+      type="submit"
+      class="button button--state-primary button--size-default"
```

Comment:
**nitpick:** We are trying to get ride of these classes as the `@apply` rule is frowned upon (apparently). Can you just use the class lists directly from flowbite.

Reply:
I'd rather leave this for now as this was just my IDE reformatting this line, vs me actually refactoring this portion.

Reply:
Sounds good